### PR TITLE
File Upload Translations

### PR DESCRIPTION
**Pull request by melloware**

### CODE REVIEW

File: primefaces/src/main/resources/org/primefaces/Messages_it.properties [hunk 1]
```diff
@@ -16,3 +16,9 @@ primefaces.password.INVALID_MATCH_detail = {0} dovrebbe corrispondere a {1}.
 primefaces.slider.OUT_OF_RANGE = {0}: Errore di validazione: Il valore è fuori intervallo.
 primefaces.tree.REQUIRED = {0}: Errore di validazione.
 primefaces.tree.REQUIRED_detail = È richiesta una selezione.
+primefaces.FileValidator.FILE_LIMIT = Numero massimo di file superato.
+primefaces.FileValidator.FILE_LIMIT_detail = Numero massimo: {0}.
+primefaces.FileValidator.ALLOW_TYPES = Tipo di file non valido.
+primefaces.FileValidator.ALLOW_TYPES_detail = Tipo di file non valido: ''{0}'. Tipi consentiti: ''{1}'.
```
Comment (NicolaIsotta) [inline primefaces/src/main/resources/org/primefaces/Messages_it.properties]: there are missing quote marks

Comment (melloware) [reply primefaces/src/main/resources/org/primefaces/Messages_it.properties]: Thanks will fix

**Comment by tandraschko** [comment]
dont forget the JS

**Comment by melloware** [comment]
Let me work on the JS thanks for the reminder @tandraschko

**Comment by melloware** [comment]
Added the JS translations!

**Pull request merged**